core/app/services/spree/credit_cards/destroy.rb

```diff
@@ -14,19 +14,15 @@ def call(card:)
       protected
 
       def invalidate_payments(card:)
-        if payment_scope(card).checkout.each(&:invalidate!)
-          success(card: card)
-        else
-          failure('invalidating payments failure')
-        end
+        payment_scope(card).checkout.each(&:invalidate!)
+
+        success(card: card)
```

Comment:
@damianlegawiec could you please look on this code? `payment_scope` contain only valid payments assigned to card so I couldn't find edge-case for return failure or exception msg